### PR TITLE
Add repo wcb

### PR DESCRIPTION
**Pull request by Pouzor**
## Description

Please explain the changes you made here and link to any relevant issues.

## Breaking Changes

Please list all the breaking changes introduced by your changes.

### Checklist

* [x] `terraform fmt` and `terraform validate` both work from the root directory (look in CI for an example)
* [x] Docs have been added/updated (for bug fixes/features)
* [x] Any breaking changes are noted in the breaking changes section above

### CODE REVIEW

File: repositories.tf [hunk 1]
```diff
@@ -138,6 +138,46 @@ module "blog_eleven-labs_com" {
...
+      status_check-contexts                      = ["deploy"]
+      pr_reviews-required_approving_review_count = 1
+      pr_reviews-require_code_owner_reviews      = false
+      pr_reviews-dismiss_stale_reviews           = false
+      pr_reviews-dismissal_users                 = [module.GORRONCharlesEric.login]
+      pr_reviews-dismissal_teams                 = []
+      restrictions-users                         = [module.GORRONCharlesEric.login]
+      restrictions-teams                         = []
+    }
+  ]
```
Comment (ch3ric) [inline repositories.tf:173]: we cannot use branches protections on private repos I think

**Review by ch3ric — COMMENTED** [review]
@Pouzor you'll probably need to sync with @VEBERArnaud before merging because this repo was already created manually

File: repositories.tf (hunk 1 above)
Comment (Pouzor) [reply repositories.tf:173]: Yep, good shoot

**Comment by Pouzor** [comment]
> 

This should work even if the repo was previously created. At leas, I didnt see something strange in the terraform output

**Review by snroki — APPROVED** [review]

**Pull request merged**